opentracing-api-extensions-tracer-spring-autoconfigure/src/test/java/io/opentracing/contrib/api/tracer/spring/autoconfigure/TracerBeanPostProcessorTest.java

```diff
@@ -73,4 +77,17 @@ public void testTracerWrapped() {
         Mockito.verify(tracerObserver).onStart(Matchers.any(SpanData.class));
     }
 
+    @Test
+    public void testGetWrappedTracerManager() {
+        assertTrue(TracerBeanPostProcessor.getManager(mockTracer) instanceof APIExtensionsTracer);
+    }
+
```

Comment:
I get a feeling that you're testing the inner workings here.
What you care about should be the postprocess functionality right?

Now I understand why you've opened up the internal method.